app/src/main/res/layout/download_confirmation.xml

```diff
@@ -20,102 +20,53 @@
-            android:drawablePadding="24dp"
-            android:fontFamily="sans-serif"
-            android:paddingStart="24dp"
-            android:paddingTop="24dp"
-            android:paddingEnd="24dp"
-            android:paddingBottom="17dp"
-            android:text="@string/downloadConfirmationReplaceOldFileText"
-            android:textColor="?attr/normalTextColor"
-            android:textSize="16sp"
-            android:textStyle="normal"
-            android:visibility="gone" />
-
-        <TextView
-            android:id="@+id/continueDownload"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:background="?attr/selectableItemBackground"
-            android:drawableStart="@drawable/ic_file_brownish_24dp"
-            android:drawablePadding="24dp"
-            android:fontFamily="sans-serif"
-            android:paddingStart="24dp"
-            android:paddingTop="24dp"
-            android:paddingEnd="24dp"
-            android:paddingBottom="17dp"
-            android:text="@string/downloadConfirmationContinue"
-            android:textColor="?attr/normalTextColor"
-            android:textSize="16sp"
-            android:textStyle="normal" />
-
-        <TextView
-            android:id="@+id/cancel"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:background="?attr/selectableItemBackground"
-            android:drawableStart="@drawable/ic_cancel_brownish_24dp"
-            android:drawablePadding="24dp"
-            android:fontFamily="sans-serif"
-            android:paddingStart="24dp"
-            android:paddingTop="17dp"
-            android:paddingEnd="24dp"
-            android:paddingBottom="24dp"
-            android:text="@string/fireCancel"
-            android:textColor="?attr/normalTextColor"
-            android:textSize="16sp"
-            android:textStyle="normal"
-            tools:ignore="UnusedAttribute" />
-    </LinearLayout>
+        android:background="?attr/selectableItemBackground"
+        android:drawablePadding="20dp"
+        android:paddingStart="16dp"
+        android:paddingTop="14dp"
+        android:paddingEnd="16dp"
+        android:paddingBottom="14dp"
+        android:text="@string/downloadConfirmationContinue"
+        android:textAppearance="@style/TextAppearance.DuckDuckGo.Body1"
+        app:drawableStartCompat="@drawable/ic_download_brownish"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/downloadMessage" />
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <TextView
+        android:id="@+id/cancel"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="?attr/selectableItemBackground"
+        android:drawablePadding="20dp"
+        android:paddingStart="18dp"
```

Comment:
should this be 16?

Reply:
No, this should be 18dp due to the icon size. Will be updated in a later iteration when we'll have a new icon.